.buildkite/commands/run-ui-tests.sh

```diff
@@ -37,7 +38,7 @@ if [[ "$TESTS_EXIT_STATUS" -ne 0 ]]; then
 fi
 
 echo "--- 📦 Zipping test results"
-cd build/results/ && zip -rq WordPress.xcresult.zip WordPress.xcresult && cd -
+cd build/results/ && zip -rq Jetpack.xcresult.zip Jetpack.xcresult && cd -
```

Comment:
My guess is that it’s generating a `WordPress.xcresult` based on the project name, so the zip command can’t find what to zip. If my guess is right, this change should fix it.

```suggestion
cd build/results/ && zip -rq Jetpack.xcresult.zip WordPress.xcresult && cd -
```